src/kernels/miopen_type_traits.hpp

```diff
@@ -76,7 +76,7 @@ struct remove_cv
     typedef typename remove_volatile<typename remove_const<T>::type>::type type;
 };
 
-#if HIP_PACKAGE_VERSION_FLAT >= 6001000000ULL && HIP_PACKAGE_VERSION_FLAT < 6001024000ULL
+#if HIP_PACKAGE_VERSION_FLAT >= 6000024000ULL && HIP_PACKAGE_VERSION_FLAT < 6001024000ULL
```

Comment:
@atamazov thanks for catching it! However, now with minor version fixed, `HIP_PACKAGE_VERSION_FLAT < 6001024000ULL` is the only place where we rely on a patch version. Is there a way to get rid of it?

https://github.com/ROCm/MIOpen/issues/2651#issuecomment-1879096776

Reply:
It might *just work* because the latest mainline is reporting `Build with HIP 6.1.24013`, just hope that patch version number does not go backwards ;-)

Reply:
@junliume 
> Is there a way to get rid of it?

No, this adapts the library for differences between 6.0, 6.0.1 and 6.1.

>  just hope that patch version number does not go backwards ;-)

If it goes, we override.